### ISSUE TITLE
Text alternative is missing for icons

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#594]
<dl><dt>Description</dt><dd>The text alternative is missing.
Alternative text is missing for all Icons on page.</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
Images that convey content MUST have programmatically-discernible alternative text.

HOW TO FIX: 
Fix this issue by using ONE of the following techniques:

1. Use the alt attribute on the <img> element to convey the intent, purpose, or meaning of the image.

<img src="sunset.jpg" alt="The sun setting over the Pacific Ocean, with a silhouette of a flying seagull in the foreground">

2. Use the aria-label attribute to convey the intent, purpose, or meaning of the image.

<p>Have a great day <span class="fonticon smiley" role="img" aria-label="Smiley face"></span></p>

REFERENCE: 
Deque University: https://dequeuniversity.com/class/images2/alt-text/informative
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/

BACKGROUND: 
People who are blind cannot see images on a page. In order to give people who cannot see an image access to the information conveyed by the image, it must have a text alternative. The text alternative must describe the information or function represented by the image. Screen readers can then use the alternative text to convey that information to the screen reader user.

For informative images - images that convey information - the alternative text must communicate the intent, purpose, or meaning of the image in a way that serves as a true alternative for the image.</dd><dt>Source code</dt><dd>

```html
<div aria-hidden="true" class="Icon Icon--add-user"><svg overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M15 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm-9-2V7H4v3H1v2h3v3h2v-3h3v-2H6zm9 4c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div>
```
</dd><dt>Checkpoint</dt><dd>Alternative Text (Informative Images) (1.1.1.b)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Icon</dd></dl>

**Comment by schne324** [comment]
this is a false positive - the icons on the docs site are decorative and the components themselves allow for accessible names to be added when necessary (and when not, they are marked with `aria-hidden=true`)
